src/usr/isteps/HBconfig

```diff
@@ -35,3 +35,15 @@ config PRINT_SYSTEM_INFO
     defaults n
     help
         Enables code that prints out the HWAS information for each target found during discovery
+
+config DYNAMIC_BIFURCATION
+    default n
+    depends on (!BARRELEYE_G2_DYNAMIC_BIFURCATION)
+    help
+        Enables code that detect PEC bifurcated config.
+
+config BARRELEYE_G2_DYNAMIC_BIFURCATION
```

Comment:
Not a fan of a system-specific config var, try to come up with a more generic name.

Reply:
will change the naming